src/yubihsm.rs

```diff
@@ -1,13 +1,10 @@
-use crate::{
```

Comment:
Seems like a lot of stuff got stuck under these `use crate::` declarations. They're really intended for modules defined within the same crate, not external crates.